pkg/config/environment_containers.go

```diff
@@ -123,7 +123,7 @@ func detectContainerd(features FeatureMap) {
 		// Containerd support was historically meant for K8S
 		// However, containerd is now used standalone elsewhere.
 		// TODO: Consider having a dedicated setting for containerd standalone
-		if IsKubernetes() {
+		if IsKubernetes() && runtime.GOOS != "windows" {
```

Comment:
Could you explain why we need this modification? 

Should we implement the TODO?

Reply:
Without the if, when the cri socket path is set, `cri` would be auto-enabled. The if is there because cri does not work under Windows.

Not sure why it wasn't there before. Enabling cri does not break anything, I think, but it shows errors because the cri check is not included in the Windows build.

Reply:
Not sure about implementing the TODO in this PR. I think that's a separate topic.
In theory, removing that `IsKubernetes()` should be enough, but I'm not sure if it can have some side-effects.

Reply:
thanks for the explanation. 
could you add a comment about we disable it on windows, and maybe create create a function `IsCRISupported()`

Reply:
I created that function and I think it's more clear now :+1: